### DISCUSSION TITLE
After Recipe Blog is installed, the relevant ".liquid" template files cannot be seen in the "Templates" of the WEB background

### DISCUSSION THREAD

**Discussion by pgy866** [OrchardCMS/OrchardCore#10416] [Q&A]
After installing the website through Recipe Blog, the related ".liquid" template file cannot be seen in "Admin/Design/Templates"
Is this the original design? Or is it missing?

通过Recipe Blog安装网站后，在“Admin/Design/Templates”看不到相关的“.liquid” 模板文件；
这个是原有设计？还是遗漏呢？
![截图_选择区域_20211006144844](https://user-images.githubusercontent.com/28694676/136153974-52e9038f-1760-453b-a9e6-d384b6e1ae00.png)

![截图_选择区域_20211006143000](https://user-images.githubusercontent.com/28694676/136153978-d521033c-2b40-4d5a-a825-edd96de509e2.png)

![截图_选择区域_20211006144328](https://user-images.githubusercontent.com/28694676/136153982-10801906-6f92-4443-b30e-d8e848913a58.png)

**Comment by deanmarcussen** [comment]
Yes, this is the intended design. The Blog Templates are included as .liquid files in the solution, but not as template files in TheAdmin.

See the Agency Recipe for an example of creating them as template files in TheAdmin
